tests/test_repository.py

```diff
@@ -157,6 +157,31 @@ def test_package_installation(container_per_test, pkg):
     )
 
 
+@pytest.mark.skipif(
+    OS_VERSION not in ALLOWED_BCI_REPO_OS_VERSIONS,
+    reason="no included BCI repository - can't test",
```

Comment:
> not included in the BCI repository

this is a skipif, there's no need to include "can't test", the logs will already produce a message like:

> SKIPPED: not included in the BCI repository

Reply:
that changes the meaning.. (it's not about it not being in the bci repo but about the bci repo not being in those containers)

Reply:
still not clear to me, but ok.

Reply:
I've removed it everywhere now (it was just a copy of all the previous messages)